samples/src/main/python/SynapsePythonSDK.py

```diff
@@ -0,0 +1,17 @@
+# This sample shows how to use the Kusto Python SDK with Synapse authentication.
+# One should use the SDK to run admin commands while using the connector to query
+# https://docs.microsoft.com/azure/data-explorer/kusto/api/python/kusto-python-client-library
+# To upload SDK package to synapse workspace, create a wheel file and follow this:
+# https://docs.microsoft.com/azure/synapse-analytics/spark/apache-spark-manage-python-packages#install-wheel-files
+from azure.kusto.data import KustoClient, KustoConnectionStringBuilder
+
+token_provider = lambda: TokenLibrary().getAccessToken("{\"audience\": \"AzureDataExplorer\", \"name\": \"\"}").getToken()
```

Comment:
this use user token in interactive mode and MSI for jobs - I would add a comment about it

Reply:
Can you add a suggestion comment with how you want to explain it 
Or add tge other way if you think it is relevant